src/main/java/org/launchcode/walkabout/models/User.java

```diff
@@ -13,20 +13,20 @@ public class User extends AbstractEntity {
     @NotNull
     private String pwHash;
 
-    public User() {}
+    private static final BCryptPasswordEncoder encoder = new BCryptPasswordEncoder();
 
     public User(String username, String password) {
         this.username = username;
         this.pwHash = encoder.encode(password);
     }
 
+    public User() {}
```

Comment:
I believe the empty constructor is support to stay above any constructors with fields initialized by it.

Reply:
You are correct. I had it backwards on my example I was using as a template. I've swapped it around.